assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.stories.js

```diff
@@ -121,6 +121,51 @@ Ready.args = {
 	},
 };
 
+export const SearchConsoleGatheringData = Template.bind( {} );
+SearchConsoleGatheringData.storyName = 'Search Console Gathering Data';
+SearchConsoleGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport( [], {
+			options: searchConsoleArgs,
+		} );
+
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export const AnalyticsGatheringData = Template.bind( {} );
+AnalyticsGatheringData.storyName = 'Analytics Gathering Data';
+AnalyticsGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
+
+export const AnalyticsAndSearchConsoleGatheringData = Template.bind( {} );
+AnalyticsAndSearchConsoleGatheringData.storyName =
+	'Analytics and Search Console Gathering Data';
+AnalyticsAndSearchConsoleGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport( [], {
+			options: searchConsoleArgs,
+		} );
+
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
+
```

Comment:
Would you please add a new story showing a non-gathering data zero data state? In the past gathering data and zero data were treated the same so we want to see these separately now 👍